packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -363,7 +458,7 @@ class VolumeViewport extends BaseVolumeViewport {
     return getClosestImageId(volume, focalPoint, viewPlaneNormal);
   };
 
-  getRotation = (): number => 0;
+  getRotation = this.getRotationGPU.bind(this);
```

Comment:
so volume viewport is always GPU, so you can use getRotation and setRotation here, stack viewport only has cpu vs gpu

Reply:
I've improved the nomenclature @sedghi